src/NuGetForUnity.Tests/Assets/Tests/Editor/NuGetTests.cs

```diff
@@ -317,21 +317,24 @@ public void InstallAndSearchLocalPackageSource([Values] bool hierarchical)
             // get the package file by installing it
             NugetPackageInstaller.InstallIdentifier(package);
             Assert.IsTrue(InstalledPackagesManager.IsInstalled(package), "The package was NOT installed: {0} {1}", package.Id, package.Version);
-            var packageFilePath = Path.Combine(
+            var nuspecFilePath = Path.Combine(
                 ConfigurationManager.NugetConfigFile.RepositoryPath,
                 $"{package.Id}.{package.Version}",
-                package.PackageFileName);
-            Assert.That(packageFilePath, Does.Exist.IgnoreDirectories);
+                package.SpecificationFileName);
+            Assert.That(nuspecFilePath, Does.Exist.IgnoreDirectories);
+
+            var nupkgFileName = $"{package.Id}.{package.Version}.nupkg";
+            var nupkgFilePath = Path.Combine(PackageCacheManager.CacheOutputDirectory, nupkgFileName);
 
```

Comment:
```suggestion
Assert.That(nupkgFilePath, Does.Exist.IgnoreDirectories);
```

Reply:
I can't see the change in the current version

Reply:
Not sure why, I added `Assert.That(nupkgFilePath, Does.Exist.IgnoreDirectories);` on line `328`, it's right after `var nupkgFilePath = Path.Combine(PackageCacheManager.CacheOutputDirectory, nupkgFileName);` if I understood the requested change correctly?

Reply:
Am I blind:
https://github.com/GlitchEnzo/NuGetForUnity/blob/501fd3437ee5b46badadfe4c40dc8f0bda0a41ad/src/NuGetForUnity.Tests/Assets/Tests/Editor/NuGetTests.cs#L326-L329

Reply:
Ok my fault